### ISSUE TITLE
Ask users of the questionnaire to help us improve it

### ISSUE THREAD

**Issue by dwsinger** [jasonanovak/security-questionnaire#12]
add to 1.1, or some other suitable place:

This questionnaire is not exhaustive. After completing a privacy review, it may be that there are privacy aspects of your specification that a strict reading, and response to, this questionnaire, would not have revealed. If this is the case, please convey those privacy concerns, and indicate if you can think of improved or new questions that would have covered this aspect.

**Comment by jasonanovak** [comment]
Good catch!  I've added it to https://github.com/jasonanovak/security-questionnaire/pull/11/commits/bfd965a628abc979afc857bc699ee68dd0af916a.
